src/components/TodoListInput.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const TodoListInput = ({ HandleAddTodo }) => {
+  return (
+    <form onSubmit={HandleAddTodo}>
```

Comment:
```suggestion
    <form onSubmit={HandleAddTodo}>
```
<form onSubmit={handleAddTodo}

Komponente nam pocinju s velikim imenom